### PR TITLE
Suirad async

### PR DESCRIPTION
**Pull request by ProgrammerDan**
Not sure what happened to your rebase, suirad; this is how it should look. Continuation of #212 , cleaned up.

### CODE REVIEW

**Comment by suirad** [comment]
https://github.com/suirad/NameLayer/commit/92a40d8fabdfb5068641f57446ccfb62666768db

**Comment by suirad** [comment]
https://github.com/suirad/NameLayer/commit/1626de303ba8f07b5992e6c5683b63f5fbf459a7

File: namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java [hunk 1]
```diff
@@ -31,42 +34,58 @@ public MercuryMessageListener() {
 	public void onMercuryMessage(AsyncPluginBroadcastMessageEvent event){
 		if (!event.getChannel().equalsIgnoreCase("namelayer"))
 			return;
-		String[] message = event.getMessage().split(" ");
+		String[] message = event.getMessage().split("|");
```
Comment (ProgrammerDan) [inline namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java]: oh this needs to be \\|

**Comment by ProgrammerDan** [comment]
Ok, here goes. After many days, delays, and way more refactoring then we would all like to admit, this PR is complete and is going into production. Thanks to @suirad for kicking it off, @Maxopoly for tons of testing and work in the middle/reviews, and @BlackXnt for testing.

Now lets hope it doesn't explode.

**Pull request merged**